### PR TITLE
chore: migrate to Owl bot

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**

### CODE REVIEW

File: owlbot.py [hunk 1]
```diff
@@ -19,18 +19,8 @@
...
-  library = gapic.node_library(name, version, proto_path=f'google/cloud/translate/{version}')
-  s.copy(library, excludes=['README.md', 'package.json', 'src/index.ts'])
-
 logging.basicConfig(level=logging.DEBUG)
 
-AUTOSYNTH_MULTIPLE_COMMITS = True
-
 common_templates = gcp.CommonTemplates()
 templates = common_templates.node_library(source_location='build/src')
 s.copy(templates, excludes=[])
```
Comment (bcoe) [inline owlbot.py:26]: we should switch to `postprocess_gapic_library_hermetic`.

File: src/index.ts [hunk 2]
```diff
@@ -11,61 +11,15 @@
...
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
 
-import * as v2 from './v2';
-
-/**
- * The `@google-cloud/translate` package has the following named exports:
- *
- * - `{@link TranslationServiceClient}` class - constructor for v3 of the Translation API.
```
Comment (bcoe) [inline src/index.ts:20]: Can we exclude one file in our copy logic somehow? This one looks to be handwritten artisanal code, I believe we will find this to be the case for several ML libraries -- they're some of the worst offenders.

File: test/index.ts [hunk 3]
```diff
@@ -1,648 +0,0 @@
...
-        translate.translate(INPUT_HTML, OPTIONS, assert.ifError);
-      });
-
-      it('should allow overriding the format', done => {
-        const options = extend({}, OPTIONS, {
-          format: 'custom format',
-        });
-
-        translate.request = (reqOpts: r.Options) => {
-          assert.strictEqual(reqOpts.json.format, options.format);
```
Comment (bcoe) [inline test/index.ts:440]: this is handwritten too.

File: owlbot.py (hunk 1 above)
Comment (SurferJeffAtGoogle) [reply owlbot.py:26]: Do you have an example of calling that function?

**Pull request closed without merging**